roles/grafana/CHANGELOG.md

```diff
@@ -0,0 +1,48 @@
+# Changelog
```

Comment:
How should we treat this changelog in the future? A separate CL apart from the collection is undesirable.

Reply:
I've already thought about removing it. There's nothing special to be found there...